src/Chapter1.hs

```diff
@@ -478,7 +478,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integral a => a -> a
+lastDigit n = abs (n `rem` 10)
```

Comment:
Nice work on using infix form of calling functions! We don't explain this syntax in our course to make it short. But well done :+1:

Reply:
Thanks!
So is ```abs (n `rem` 10)``` the same as ```(abs n) `mod` 10``` ?

I was a bit surprised that ```(-12) `mod` 10 == 8```

Reply:
Yes, this is absolutely correct 💯